### PR TITLE
Publish changes

### PR DESCRIPTION
**Pull request by snopoke**
@czue 

buddy @orangejenny

### CODE REVIEW

File: corehq/form_processor/change_publishers.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def change_meta_from_sql_case(case):
         document_type='CommCareCaseSql',  # todo: should this be the same as the couch models?
         document_subtype=case.type,
         domain=case.domain,
-        is_deletion=False,
+        is_deletion=case.is_deleted,
```
Comment (czue) [inline corehq/form_processor/change_publishers.py]: :+1:

**Comment by czue** [comment]
nice looks good. glad you didn't need to touch any pillow code too. :+1: on :white_check_mark:

**Comment by snopoke** [comment]
probably would be good to have tests for this. hold of on merge till I add them

File: testapps/test_pillowtop/utils.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
+import uuid
+
+from decorator import ContextManager
```
Comment (dimagimon) [inline testapps/test_pillowtop/utils.py]: F401 'ContextManager' imported but unused

File: corehq/ex-submodules/couchforms/tests/test_archive.py [hunk 3]
```diff
@@ -1,13 +1,18 @@
 import os
 from datetime import datetime, timedelta
 from django.test import TestCase
+from django.test.testcases import SimpleTestCase
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/tests/test_archive.py]: F401 'SimpleTestCase' imported but unused

**Comment by snopoke** [comment]
Forgot about this. Updated with tests

File: testapps/test_pillowtop/utils.py [hunk 4]
```diff
@@ -28,3 +32,23 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         settings.PILLOWTOPS = self._PILLOWTOPS
+
+
+class capture_kafka_changes_context(object):
```
Comment (czue) [inline testapps/test_pillowtop/utils.py]: this is nifty. :gift:

**Comment by czue** [comment]
look great, thanks!

**Pull request merged**